.eslintrc

```diff
@@ -1,6 +1,10 @@
 {
   "extends": ["4catalyzer", "prettier"],
   "plugins": ["prettier"],
+  "globals": {
```

Comment:
env browser?

Reply:
actually I'd just relax the rule in the executeScript block